style.css

```diff
@@ -70,3 +71,250 @@ body {
   list-style: none;
   padding: 0;
 }
+
+.scroll {
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+  align-items: center;
+  gap: 30px;
+  width: 100%;
+  padding: 100px 0;
+  background-color: #f7f7f9;
+}
```

Comment:
Please ensure that the main container of the projects has a display: grid according to project requirements.